src/amo/components/Addon/index.js

```diff
@@ -173,69 +193,127 @@ export class AddonBase extends React.Component {
         header={i18n.gettext('Rate your experience')}
         className="Addon-overall-rating"
         {...props}>
-        <RatingManager
-          addon={addon}
-          location={location}
-          version={addon.current_version}
-        />
+        {addon ?
+          <RatingManager
```

Comment:
Actually, yeah, I was thinking about this too. The 'No reviews yet' is an answer to a question asked, which is 'how many reviews are there for this add-on?' In this case, it is not the right answer to give. I'll see if I can change it without much churn.